paramiko/transport.py

```diff
@@ -17,6 +17,7 @@
 # along with Paramiko; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA.
 
+# flake8: noqa
```

Comment:
a bunch of changes were made below to appease flake8, is this a leftover from partway through the process?